MCprep_addon/spawner/meshswap.py

```diff
@@ -754,10 +759,10 @@ def execute(self, context):
 		elif self.runcount == 1:
 			self.report({'INFO'}, "Swapped 1 object")
 			return {'FINISHED'}
-		self.report({'INFO'}, "Swapped {} objects".format(self.runcount))
+		self.report({'INFO'}, f"Swapped {self.runcount} objects")
 		return {'FINISHED'}
 
-	def prep_obj_list(self, context):
+	def prep_obj_list(self, context: Context) -> List[Object]:
```

Comment:
Replace `Object` with `bpy.types.Object`
```suggestion
	def prep_obj_list(self, context: Context) -> List[bpy.types.Object]:
```